XCUITests/IntegrationTests.swift

```diff
@@ -55,7 +55,7 @@ class IntegrationTests: BaseTestCase {
     private func waitForInitialSyncComplete() {
         navigator.nowAt(BrowserTab)
         navigator.goto(SettingsScreen)
-        waitForExistence(app.tables.staticTexts["Sync Now"], timeout: 15)
+        waitForExistence(app.tables.staticTexts["Sync Now"], timeout: 25)
```

Comment:
Do we know why recently its needed to bump a timeout incase we need to file a regression?

LGTM

Reply:
I just increased to be sure the option is there...if it is there, we don't wait until the end...so tests will not be impacted. Thanks!